scripts/github-changelog.cr

```diff
@@ -163,11 +165,17 @@ sections = array.group_by { |pr|
   end || "Other"
 }
 
-titles = sections.keys.sort!
+titles = [] of String
+["Compiler", "Language", "Standard Library", "Tools", "Other"].each do |main_section|
```

Comment:
I guess this could be changed to `["Language", "Standard Library", "Compiler", "Tools", "Other"]` because language and stdlib are more impactful to the average user than compiler changes?